### PR TITLE
databricks

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**Are you a current Fivetran customer?** 

fivetran made PR

**What change(s) does this PR introduce?** 

ensures databricks compatibility via buildkite. this is dependent on https://github.com/fivetran/dbt_fivetran_utils/pull/98

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature https://github.com/fivetran/dbt_netsuite/issues/55
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧱 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -56,4 +56,18 @@ steps:
             - "CI_REDSHIFT_DBT_PASS"
             - "CI_REDSHIFT_DBT_USER"
     commands: |
-      bash .buildkite/scripts/run_models.sh redshift
\ No newline at end of file
+      bash .buildkite/scripts/run_models.sh redshift
+
+  - label: ":databricks: Run Tests - Databricks"
```
Comment (fivetran-avinash) [inline .buildkite/pipeline.yml:61]:
```suggestion
  - label: ":bricks: Run Tests - Databricks"
```

Don't think there's a :databricks: emoji =P

**Review by fivetran-avinash — CHANGES_REQUESTED** [review]
@fivetran-jamie Mostly good! Some changes but mostly syntax.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,16 @@ To use this dbt package, you must have At least either one Fivetran **Netsuite**
 - vendorcategory
 
 ### Database Compatibility
-This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, or **Databricks** destination.
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:71]: ah yeah so this step is formatted differently due to the Netsuite vs Netsuite2 section

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,9 @@
 packages:
-- package: fivetran/fivetran_utils
-  version: [">=0.4.0", "<0.5.0"]
+# - package: fivetran/fivetran_utils
+#   version: [">=0.4.0", "<0.5.0"]
+- git: https://github.com/fivetran/dbt_fivetran_utils.git
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply packages.yml:4]: yep fivetran_utils v0.4.2 will have to be released beforehand!

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (fivetran-jamie) [reply .buildkite/pipeline.yml:61]: isn't it this? 
image

Comment (fivetran-avinash) [reply .buildkite/pipeline.yml:61]: Will it show up in Github though? Typing :databricks: here just leaves :databricks:

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply README.md:71]: Oops I didn't read up.

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (fivetran-avinash) [reply .buildkite/pipeline.yml:61]: ignore, this is for Buildkite, where it works fine =P

**Review by fivetran-avinash — APPROVED** [review]
LGTM!

**Pull request merged**